File: Array&String/uncommon_words_from_two_sentences.py
from collections import Counter

class Solution:
    def uncommonFromSentences(self, s1: str, s2: str) -> list[str]:
        cs = Counter((s1 + " " + s2).split())
        ans = []
        for k in cs:
            if cs[k] == 1:
                ans.append(k)
        return ans

File: Array&String/CPP/uncommon_words_from_two_sentences.cpp
#include <map>
#include <sstream>
#include <string>
#include <vector>
using namespace std;

class Solution {
public:
    vector<string> uncommonFromSentences(string s1, string s2) {
        istringstream s(s1 + " " + s2);
        map<string, int> cs;
        while (s >> s1) cs[s1]++;
        vector<string> res;
        for (auto& k: cs) {
            if (k.second == 1 ) res.push_back(k.first);
        }
        return res;
    }
};